lib/grpramat.gi

```diff
@@ -251,7 +251,9 @@ function( G )
 
     # if not rational, use the nice monomorphism into a rational matrix group
     if not IsRationalMatrixGroup( G ) then
-        return IsFinite( Image( NiceMonomorphism( G ) ) );
+        # the following does not use NiceObject(G) as the only method for
+        # that currently requires IsHandledByNiceMonomorphism
+        return IsFinite( Image( NiceMonomorphism( G ), G ) );
```

Comment:
Why not `IsFinite(NiceObject(G))` ?

Reply:
As explained in the comment right above this line, there is no method to compute `NiceObject` in this case. We could also change the existing `NiceObject` to apply to arbitrary groups, not just those in the filter `IsHandledByNiceMonomorphism`, but that's a bigger change with potentially further implications, and so I decided to go this way, as it minimizes the risk of a regression. One could investigate the mentioned alternative in a separate PR, but I'd rather not mix it with a bugfix.